modules/alert_policy/alert_policy.tf

```diff
@@ -0,0 +1,52 @@
+variable "alert_policy" {
```

Comment:
So just as a general pattern question -- why don't we put this in `variables.tf`?

And while we're at it, why do we we put this stuff in `alert_policy.tf` instead of `main.tf`? Just curious mostly..

Reply:
For consistency across our modules
https://github.com/cloudposse/terraform-datadog-monitor/tree/master/modules/cpu

I'm easy with both approaches and as `main.tf` doesn't have anything other than provider maybe we should change the convention and put everything in `main.tf`.

Reply:
I see.. I'm somewhat impartial but perhaps @osterman can chime in.

Reply:
oh,`terraform-datadog-monitor` is probably the *worst* example to use =) it was never used in production, still on HCLv1, and has no tests. I don't believe any of our HCL2 modules with tests do this. 

By in large, we do try to follow the hashicorp recommended best practices for layouts - and if we don't let me know and we'll try to correct it.

Reply:
Reorganized repo.